packages/licenses/src/getPkgInfo.ts

```diff
@@ -23,7 +28,17 @@ export async function readPkg (pkgPath: string) {
  * @const
  * List of typical names for license files
  */
-const LICENSE_FILES = ['LICENSE', 'LICENCE', 'LICENSE.md', 'LICENCE.md']
+const LICENSE_FILES = [
+  'LICENSE',
+  'LICENCE',
+  'LICENSE.md',
+  'LICENCE.md',
+  'LICENSE.txt',
+  'LICENCE.txt',
+  'MIT-LICENSE.txt',
+  'MIT-LICENSE.md',
+  'MIT-LICENSE',
+]
```

Comment:
We have also code that searches for licenses in the pack command. We should be able to reuse the same code in both places:

https://github.com/pnpm/pnpm/blob/a4c58d424a69225f4031e9935abdbeeb9ef3a80d/packages/plugin-commands-publishing/src/pack.ts#L19-L20

Reply:
I will have a look, can I make this change as part as a second PR? At first site it looks like this is scanning a directory instead of the package file index object

Reply:
ok